python/cudf/cudf/io/parquet.py

```diff
@@ -908,12 +908,20 @@ def _read_parquet(
                 "cudf engine doesn't support the "
                 f"following positional arguments: {list(args)}"
             )
-        return libparquet.read_parquet(
-            filepaths_or_buffers,
-            columns=columns,
-            row_groups=row_groups,
-            use_pandas_metadata=use_pandas_metadata,
-        )
+        if cudf.get_option("mode.pandas_compatible"):
+            return libparquet.ParquetReader(
```

Comment:
By default `chunk_read_limit=0 (meaning no limit)` so would we ever end up with more than 1 chunk case?

Reply:
Yes, there are multiple chunks being generated. Since `pass_read_limit` is `1024000000`